src/main/java/com/exasol/releasedroid/adapter/jira/JiraReleaseMaker.java

```diff
@@ -55,16 +53,14 @@ private String createTicketRequest(final Repository repository) throws JiraExcep
     }
 
     private String getHumanReadableName(final Repository repository) {
-        final String projectNameFromConfig = getProjectNameFromConfig(repository);
-        if (projectNameFromConfig != null) {
-            return projectNameFromConfig;
-        } else {
-            return getDefaultProjectName(repository.getName());
-        }
+        final Optional<String> projectNameFromConfig = getProjectNameFromConfig(repository);
+        return projectNameFromConfig.orElseGet(() -> getDefaultProjectName(repository.getName()));
```

Comment:
Not 100%, bu I think this can be simplified to:
```suggestion
        return projectNameFromConfig.orElse(getDefaultProjectName(repository.getName()));
```

Reply:
`orElse` takes a variable/constant and `orElseGet` takes a Supplier(lambda function). So in this case I can't use `orElse` as I call a function from it.

Reply:
Hm, I think I see my mistake. orElse() would always evaluate the expression (always would call getDefaultProjectName(...)), even if the Optional is not empty.